praw/config.py

```diff
@@ -50,13 +51,13 @@ def _load_config(cls):
         cls.CONFIG = config
 
     @property
-    def short_url(self):
+    def short_url(self) -> str:
         """Return the short url or raise a ClientException when not set."""
         if self._short_url is self.CONFIG_NOT_SET:
             raise ClientException("No short domain specified.")
         return self._short_url
 
-    def __init__(self, site_name, **settings):
+    def __init__(self, site_name: str, **settings: Union[str, Any]):
```

Comment:
Is this union necessary with `Any`?

Reply:
I think that multiple types of information can be provided, but if it's 100% strings, I can remove it.